static/app/views/issueDetails/groupReplays/groupReplays.tsx

```diff
@@ -19,9 +17,16 @@ type Props = {
   group: Group;
 };
 
+const VISIBLE_COLUMNS = [
+  ReplayColumns.replay,
+  ReplayColumns.os,
+  ReplayColumns.browser,
+  ReplayColumns.duration,
+  ReplayColumns.countErrors,
+  ReplayColumns.activity,
+];
+
 function GroupReplays({group}: Props) {
-  const theme = useTheme();
-  const hasRoomForColumns = useMedia(`(min-width: ${theme.breakpoints.small})`);
```

Comment:
How does this look at small breakpoints now?

Reply:
I'll take some screen shots!